packages/compass-connections-navigation/src/database-item.tsx

```diff
@@ -92,18 +48,25 @@ export const DatabaseItem: React.FunctionComponent<
 > = ({
   id,
   name,
+  level,
   posInSet,
   setSize,
   isExpanded,
   isActive,
   isReadOnly,
+  isLegacy,
```

Comment:
Maybe there is a better name for this taking into account that we will need to support this in shared code path for a single connection in compass-web use-case